doc/api/net.md

```diff
@@ -583,259 +772,270 @@ the socket is destroyed (for example, if the client disconnected).
+`'IPv4'` or `'IPv6'`.
 
 ### socket.remotePort
 <!-- YAML
 added: v0.5.10
 -->
 
-The numeric representation of the remote port. For example,
-`80` or `21`.
+Provides the numeric representation of the remote port. For example, `80` or
+`21`.
 
 ### socket.resume()
 
-Resumes reading after a call to [`pause()`][].
+The `socket.resume()` method resumes reading after a call to
+[`socket.pause()`][]. See the [Readable][] documentation for more information.
 
 ### socket.setEncoding([encoding])
 <!-- YAML
 added: v0.1.90
 -->
 
-Set the encoding for the socket as a [Readable Stream][]. See
-[`stream.setEncoding()`][] for more information.
+The `socket.setEncoding()` method sets the encoding for the socket as a
+[Readable Stream][]. See [`readable.setEncoding()`][] for more information.
 
 ### socket.setKeepAlive([enable][, initialDelay])
 <!-- YAML
 added: v0.1.92
 -->
 
-Enable/disable keep-alive functionality, and optionally set the initial
-delay before the first keepalive probe is sent on an idle socket.
-`enable` defaults to `false`.
+* `enable` {boolean} `true` to enable keep-alive, `false` to disable. Defaults
+  to `false`.
+* `initialDelay` {number} The number of milliseconds to set the delay between
+  the last data packet received and the first keepalive probe. Setting `0` for
+  `initialDelay` will leave the value unchanged from the default or previous
+  setting. Defaults to `0`.
 
-Set `initialDelay` (in milliseconds) to set the delay between the last
-data packet received and the first keepalive probe. Setting 0 for
-initialDelay will leave the value unchanged from the default
-(or previous) setting. Defaults to `0`.
+The `socket.setKeepAlive()` method enables or disables keep-alive functionality,
+and optionally sets the initial delay before the first keepalive probe is sent
+on an idle socket.
 
-Returns `socket`.
+Returns a reference to `socket`.
 
 ### socket.setNoDelay([noDelay])
 <!-- YAML
 added: v0.1.90
 -->
 
-Disables the Nagle algorithm. By default TCP connections use the Nagle
-algorithm, they buffer data before sending it off. Setting `true` for
-`noDelay` will immediately fire off data each time `socket.write()` is called.
-`noDelay` defaults to `true`.
+* `noDelay` {boolean} If `true`, disables Nagle's algorithm. Defaults to
+  `true`
 
-Returns `socket`.
+The `socket.setNoDelay()` method enables or disables Nagle's algorithm. By
+default TCP connections use Nagle's algorithm and buffer data before sending
+it off. Setting `true` for `noDelay` will immediately fire off data each time
+`socket.write()` is called.
+
+Returns a reference to `socket`.
 
 ### socket.setTimeout(timeout[, callback])
 <!-- YAML
 added: v0.1.90
 -->
 
-Sets the socket to timeout after `timeout` milliseconds of inactivity on
-the socket. By default `net.Socket` do not have a timeout.
+* `timeout` {number}
+* `callback` {Function}
+
+The `socket.setTimeout()` method sets the socket to timeout after `timeout`
```

Comment:
sets the socket to "time out" maybe? I'm ESL but from what I remember "timeout" as a word is only for a sports game break as a noun.